frontend/src/pages/chat/components/chat-layout/chat-layout.tsx

```diff
@@ -34,20 +32,32 @@ const ChatLayout: React.FC = () => {
 
   const handleSend = useCallback(
     ({ message }: ChatInputValue): void => {
-      if (!hasId && currentChatMessages.length === EMPTY_ARRAY_LENGTH) {
+      if (!hasId || currentChatMessages.length === EMPTY_ARRAY_LENGTH) {
         void dispatch(chatActions.createChat({ message }));
+      } else {
+        void dispatch(
+          chatActions.createMessage({
+            message,
+            chatId: id as string,
+          }),
+        );
       }
-      // TODO: dispatch redux action to send message
     },
-    [dispatch, currentChatMessages.length, hasId],
+    [dispatch, currentChatMessages.length, hasId, id],
   );
 
+  useEffect(() => {
+    if (id) {
+      void dispatch(chatActions.getCurrentChatMessages(id.toString()));
```

Comment:
```suggestion
      void dispatch(chatActions.getCurrentChatMessages(id));
```
Isn't it already a string?